### ISSUE TITLE
404 page

### ISSUE THREAD

**Issue by SMD-1** [shreya024/MemoriesApp#9]
Create the 404 page

**Comment by Ahad1317** [comment]
I will work on it with my best efforts, Can you please assign me.

**Comment by fykaa** [comment]
@Ahad1317 You are assigned for the issue namely 404 Error Page. You may start contributions🎉

**Comment by shreya024** [comment]
> I will work on it with my best efforts, Can you please assign me.

@Ahad1317 Any updates?

**Comment by soumali28** [comment]
@Ahad1317 any updates on the 404 page?

**Comment by Ahad1317** [comment]
![Screenshot 2022-08-14 201652](https://user-images.githubusercontent.com/96586030/184542301-5c899e37-f2fa-45fc-9380-f6176d0cd9a3.png)

@soumali28 @shreya024 sorry for delay, Heres a design for the 404 page, Tell me if changes have to be done, in font, color, or anything

**Comment by shreya024** [comment]
@Ahad1317 Instead of SORRY,change the test to OOPS,looks like you are lost :( and for the image try something like this https://www.freepik.com/free-vector/lost-concept-illustration_19468134.htm#query=lost%20map&position=0&from_view=keyword

**Comment by Ahad1317** [comment]
![image](https://user-images.githubusercontent.com/96586030/184546593-c330f9e7-b492-4ee6-a953-d22df32da6f2.png)

Opinion ?

**Comment by shreya024** [comment]
> ![image](https://user-images.githubusercontent.com/96586030/184546593-c330f9e7-b492-4ee6-a953-d22df32da6f2.png)
> 
> Opinion ?

Keep the background color and text color same as before

**Comment by Ahad1317** [comment]
![image](https://user-images.githubusercontent.com/96586030/185192322-a1088bc0-1a5c-4da1-bb25-ed52988ba9d0.png)

@shreya024

**Comment by shreya024** [comment]
Perfect @Ahad1317.You may make a PR

**Comment by shreya024** [comment]
@Ahad1317 Any updates?

**Comment by Ahad1317** [comment]
Yeah Actually I was having some issues making the PR. Got it cleared from the mentor, I will make it asap now.

**Comment by shreya024** [comment]
> Yeah Actually I was having some issues making the PR. Got it cleared from the mentor, I will make it asap now.

Any updates?

**Comment by soumali28** [comment]
@Ahad1317 any updates regarding the pr. Please do it asap else it will be considered for reassignment.

**Comment by Himanshu-370** [comment]
Assign me 
Lemme try

**Comment by shreya024** [comment]
> Assign me Lemme try

Any updates?

**Comment by Harshwardhan-Patil** [comment]
Hi @shreya024 ,I'd like to work on this

**Comment by Himanshu-370** [comment]
> > Assign me Lemme try
> 
> Any updates?

I am sorry
I ll submit a PR by tonight
